docs/github-install.rst

```diff
@@ -96,6 +96,7 @@ Github action example
           uses: fjogeleit/http-request-action@master
           with:
             method: 'POST'
+            # The URL below might need to be updated since the warehouse is being renamed to app store
```

Comment:
This needs to be changed but coordinate the rename here with the install of the renamed app on tethys-staging